src/create.js

```diff
@@ -1,38 +1,56 @@
+import post from './module/post';
+
 export default function Scores() {
```

Comment:
- [ ] For best practices and as required for the project, kindly make sure you use the **arrow functions** instead of the `function` keyword.